api/pom.xml

```diff
@@ -10,24 +10,24 @@
     <parent>
         <groupId>com.flipkart.poseidon</groupId>
         <artifactId>poseidon</artifactId>
-        <version>3.1.1</version>
+        <version>4.0.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>api</artifactId>
-    <version>3.1.1</version>
+    <version>${parent.version}</version>
```

Comment:
Just like groupId, we can ignore version too for child module. It should use the parent version